Milestone 1: Basic project structure

RaminMammadzada opened the pull request:
- Initialize app by using `create-react-app`
- Delete any unnecessary files and assets created automatically by `create-react-app`
- Import prop-types and big.js
- Used `create-react-app-buildpack` to deploy the app to Heroku.

RaminMammadzada committed:
Remove unnecessary files, import big.js, and prop-types.

RaminMammadzada committed:
Add Eslint and Stylelint config files. Update codebase to fix issues shown by linters.

RaminMammadzada committed:
Update README.

mikenath223 commented:
### STATUS: INVALID PR ❌

This PR has been marked as invalid.

Reason: 
- Please enable GitHub actions and ensure all linting issues are checked and fixed.

Feel free to contact me on slack(@michgolden) if you need further clarifications.

RaminMammadzada committed:
Update linter file location for Github Action.

RaminMammadzada commented:
> ### STATUS: INVALID PR ❌
> This PR has been marked as invalid.
> 
> Reason:
> 
> * Please enable GitHub actions and ensure all linting issues are checked and fixed.
> 
> Feel free to contact me on slack(@michgolden) if you need further clarifications.

I didn't locate the linter config file for GitHub actions. I fixed it now.

christianotieno reviewed (approved):
## Milestone Approved 🍾 
Hello @RaminMammadzada 👋 
##
This milestone has now met all the requirements needed for approval. 👏 
Congratulations on the hard work done with the changes and overall, on this milestone. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy Coding 💻 
#### Christian Otieno (CR)
**| [Twitter](https://twitter.com/iamchrisotieno) | [Github](https://github.com/christianotieno) | [LinkedIn](https://www.linkedin.com/in/christianotieno/) |**
